### DISCUSSION TITLE
Where can I find my IP-Adress

### DISCUSSION THREAD

**Discussion by Kreijeck** [NebulaModTeam/nebula#383] [Q&A] [answered]
Hello all,
at first very nice idea with the multiplayer mod! Thanks too all developer!

I want to test and play the mod with a friend, but unfortunatly I am not so familiar in network topics. I hope someone can help me here.
Where can I find out my IP-address? So my friend knows to which IP he should connect.

Unfortunatly I see with ipconfig only an ipv6-adress. Does the mod works with that as well or do I need a ipv4-address?

Thanks and Regards
Thomas

**Comment by hubastard** [comment, marked as answer]
You can get your IP from this site: https://www.myexternalip.com/

And yes we support ipv4 and ipv6

**Reply by Kreijeck** [reply]
Hi again, 
thanks for your help.
Today I want to try it.
I make all portforwarding (in the Window-Firewall and the Router).
Also we try it with the ipv6 and ipv4-adress.

Unfortunatly we can't connect to each other.

Are there any further steps necessary? Are there any logs helpful that could be sent from me?

**Reply by PhantomGamers** [reply]
> Are there any further steps necessary? Are there any logs helpful that could be sent from me?

You can check https://portchecker.co/ while the server is running to make sure that you opened your port correctly

**Reply by Kreijeck** [reply]
Thanks. Yes that port is closed. So I have to take a look why

**Reply by Kreijeck** [reply]
Thanks, now it works. It was really a port problem with my router.
And now I have a lot of fun with my friends here in multiplayer!!

If I can support you with any logs or something like this, if we find some bugs please let me know!
